UPGRADE-3.1.md

```diff
@@ -24,6 +24,40 @@ FrameworkBundle
 
  * As it was never an officially supported feature, the support for absolute
    template paths has been deprecated and will be removed in Symfony 4.0.
+ * The core form types registered as services:
```

Comment:
Done. I've first went for @xabbuh suggestion but it was to weird to have just the list.

Since this is the upgrade file and might be relevant to ones getting deprecation notices, so a minimum of what and what to do make sense IMHO so I use @fabpot's.

It's now accorded to the framework bundle changelog. Final call ?